livesync/indico_livesync/handler.py

```diff
@@ -86,6 +87,8 @@ def _created(obj, **kwargs):
         parent = obj.event
     elif isinstance(obj, SubContribution):
         parent = obj.contribution
+    elif isinstance(obj, EventNote):
+        parent = obj.event if isinstance(obj.object, Session) else obj.object
```

Comment:
I think we need to have a closer look at this - probably we do not need to record a change for the parent object of the note when only the note was created/changed. Because in the end all we need to send is the note itself, isn't it?

Reply:
Haven't seen this before. Yes, at least for now, the event record is not aware of any notes. So I would also say it's fine to use the note itself.